thanos/Chart.yaml

```diff
@@ -9,7 +9,7 @@ keywords:
 sources:
   - https://github.com/thanos-io/thanos
   - https://github.com/banzaicloud/banzai-charts/tree/master/thanos
-version: 0.3.6
+version: 0.3.7
```

Comment:
No need to update the version I'm planning to release the last 3 commit as 0.3.6